liwords-ui/src/gameroom/table.tsx

```diff
@@ -209,9 +201,7 @@ export const Table = React.memo((props: Props) => {
           }
         )
         .then((streakresp) => {
-          if (stillMountedRef.current) {
-            setStreakGameInfo(streakresp.data.game_info);
-          }
+          setStreakGameInfo(streakresp.data.game_info);
```

Comment:
this one would probably need to be protected as it's an XHR - what do you think?

Reply:
it is. we're no longer using `React.useState` directly